cics-bundle-maven-plugin/src/main/java/com/ibm/cics/cbmp/AbstractCICSBundleMojo.java

```diff
@@ -53,7 +59,7 @@
 	protected String classifier;
 
 	@Parameter(required = false)
-	protected List<BundlePart> bundleParts = Collections.emptyList();
+	protected List<BundlePartBinding> bundleParts = Collections.emptyList();
 
 	@Parameter(defaultValue = "MYJVMS", required = false, readonly = false)
 	protected String defaultjvmserver;
```

Comment:
Can defaultjvmserver be private now you have a getter? And maybe final?

Reply:
Probably can't be final because of Maven, but can probably be private! Thanks.